### PR TITLE
feat(strings): add is palindrome example for C#

### PR DESCRIPTION
**Pull request by ming-tsai**
Hi @akrish4 or @Arsenic-ATG ,
Please check it out

### CODE REVIEW

**Comment by akrish4** [comment]
I dont know c# language 
I am sorry @ming-tsai 
@Arsenic-ATG   @paulsonjpaul  kindly do help him

File: strings/README.md [hunk 1]
```diff
@@ -3,3 +3,7 @@
 ### C or C++
 
 1. [Palindrome Check](c-or-cpp/palindrome.c)
+
+### C#
+Please use [.Net Finddle](https://dotnetfiddle.net/)
```
Comment (Arsenic-ATG) [inline strings/README.md:8]: Shouldn't it be " you can use any online IDE (for example [.net Finddle](https://dotnetfiddle.net/) ) to test them " ?

**Review by Arsenic-ATG — APPROVED** [review]
Algorithm is good and is working as intended. 

A suggestion is written as comment.

**Review by paulsonjpaul — COMMENTED** [review]
Hi. I don't know C#, but by reading the code I think you are not converting the input to alpha-numeric words. According to Wikipedia:

> Sentence-length palindromes ignore capitalization, punctuation, and word boundaries

So, if I put `Mr. Owl ate my metal worm` as the input, it would return `false`!

It would be nice if you strip the punctuation, white spaces, etc. and convert them into same case (lower or upper).

File: strings/README.md (hunk 1 above)
Comment (ming-tsai) [reply strings/README.md:8]: Ok

**Comment by ming-tsai** [comment]
Hi @paulsonjpaul and @Arsenic-ATG,
Check it again

**Review by paulsonjpaul — APPROVED** [review]

**Review by Arsenic-ATG — COMMENTED** [review]
looks good 👌

**Pull request merged**